Gemfile

```diff
@@ -1,62 +1,71 @@
-source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+source "https://rubygems.org"
 
-ruby '3.0.0'
+ruby "3.3.0"
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
-gem 'rails', '~> 7.0', '>= 7.0.4'
-
-# Use Puma as the app server
-gem 'puma', '~> 5.0'
-# Use SCSS for stylesheets
-gem 'sass-rails', '>= 6'
-# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
-gem 'webpacker', '~> 5.0'
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem 'turbolinks', '~> 5'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.7'
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.1.3"
 
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem "sprockets-rails"
```

Comment:
I think you will also want the `sprockets` gem 🤔 

![Screenshot 2024-02-06 at 11 39 11 AM](https://github.com/clerk/clerk-rails-starter/assets/69559/66390cda-2a28-4f81-b2ad-4b876e36def2)

https://guides.rubyonrails.org/asset_pipeline.html#what-is-the-asset-pipeline-questionmark